### ISSUE TITLE
[Bug]: [BAD_HTTP_STATUS: 403] Potential causes: IP block or streaming URL deciphering failed #6701

### ISSUE THREAD

**Issue by ChipSendype** [FreeTubeApp/FreeTube#6855]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have [searched the issue tracker for open and closed issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.

### Describe the bug

All videos will not play.  Message "[BAD_HTTP_STATUS: 403] Potential causes: IP block or streaming URL deciphering failed". 
 
Problem started Feb. 20th, 2025 and persists. Tried Ctrl-r refresh page, worked 1 in 5 times, doesn't work at all now.  Same behavior using VPN as not. 

### Expected Behavior

I expect the video to play. 

### Issue Labels

accessibility issue

### FreeTube Version

v0.23.1 Beta

### Operating System Version

Linux Mint 21.3 Virginia     base: Ubuntu 22.04 jammy

### Installation Method

.deb

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

v0.21

### Additional Information

The problem has intermittently occurred for months, but been sorted in a couple days usually. 
Always been able to find a work around/repair/setting change, 
but this problem has none and Freetube is completely unusable. 
Love what you guys do and I realize the behemoth you're up against.
It's not like I can't just use Youtube's page with a VPN, so I'm not too bummed and patiently await your repairs. Godspeed! 

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Duplicate of #6701
